src/arrayMethodReduce.js

```diff
@@ -5,7 +5,21 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let res = initialValue;
```

Comment:
You should write full names of variables for better understanding

Reply:
agree, create better name for a variable